src/service/AdminService.java

```diff
@@ -0,0 +1,550 @@
+        Set<Loan> emptyList = new HashSet<>();
+
+        if (companyId != 0){
+            Customer currentCustomer = new Customer(firstName, lastName, email, password, companyId, emptyList, address);
+            customers.add(currentCustomer);
+//            System.out.println(currentCustomer);
+        }
+        else{
+            System.out.println("There is no company with the name provided!");
+        }
+    }
+
+    private void removeBookFromList(int idToRemove){
+        books.removeIf(x -> x.getId() == idToRemove);
+    }
+
+    private void removeAudioBookFromList(int idToRemove){
+        audiobooks.removeIf(x -> x.getId() == idToRemove);
+    }
+
+    private void removeEBookFromList(int idToRemove){
+        ebooks.removeIf(x -> x.getId() == idToRemove);
+    }
+
+    private void removeCompany(int idToRemove){
+        companies.removeIf(x -> x.getCompanyId() == idToRemove);
+    }
+
+    private void displayBooks(){
+        BookNameComparator bookNameComparator = new BookNameComparator();
+        books.sort(bookNameComparator);
+
+        for (Book book : books){
+            System.out.println(book);
+        }
+    }
+
+    private void displayAudiobooks(){
+        for (Audiobook audiobook : audiobooks){
+            System.out.println(audiobook);
+        }
+    }
+
+    private void displayEBooks(){
+        for (EBook ebook : ebooks){
+            System.out.println(ebook);
+        }
+    }
+
+    private void displayCompanies(){
+        for (Company company : companies){
+            System.out.println(company);
+        }
+    }
+
+    public void displayCustomers(){
+        for (Customer customer : customers){
+            System.out.println(customer);
+        }
+    }
+
+    @Override
+    public void menu(Scanner scanner, AdminService admin, String username) {
+        System.out.println("\n\t\t\t------------- ADMIN MENU -------------");
+
+        while(true){
+            System.out.println("\t Please choose what you want to do: ");
+            System.out.println("\t -> Option 1 - Add items in the database ");
+            System.out.println("\t -> Option 2 - Remove items in the database ");
+            System.out.println("\t -> Option 3 - Display existing items ");
+            System.out.println("\t -> Option 4 - EXIT ");
+
+            int option1 = scanner.nextInt();
+
+            switch (option1) {
+                case (1):
+                    // Add
+
+                    System.out.println("\t What type of item do you want to add?");
+                    System.out.println("\t -> Option 1 - Book ");
+                    System.out.println("\t -> Option 2 - AudioBook ");
+                    System.out.println("\t -> Option 3 - EBook ");
+                    System.out.println("\t -> Option 4 - Company ");
+
+                    int option2 = scanner.nextInt();
+                    scanner.nextLine();
+
+                    switch (option2) {
+                        case (1) -> addBook(scanner);
```

Comment:
when running the program thiis does not work, ":" expected - does your program run on your local machine? which java version are you using?

Reply:
is a new option for case from java 12 I can see and since I have java 11 on my laptop that's why it does not run  :) 👍

Reply:
Yes, it runs on my local machine :) Do I need to change it so it will run on your machine?

Reply:
no, you do not need to change it if you are using jdk17